CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## [1.2] - 2024-01-22
```

Comment:
Does this need changes to workflow at all? If not, then 1.2 is a good version number, but if this change is not compatible and would need passari-workflow changes too, then 2.0...

Reply:
This doesn't require changes in workflow